study2/index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Progate</title>
+    <link rel="stylesheet" href="stylesheet.css">
+  </head>
+  <body>
+    <!-- ここからHTMLを書き始めてください -->
+    <header>
+      <div class="header-container">
+        <div class="logo">
+          Progate
+        </div>
+        <div class="menu-list">
```

Comment:
セマンティックな記述にするために基本的には header のナビゲーションは nav タグを使いましょう。

```suggestion
        <nev class="global-nav">
```